source/api_documentation/call_ingestion_api/index.rst

```diff
@@ -324,7 +324,7 @@ Supported Recording Formats
 ---------------------------
 
 The default call recording format on the Invoca platform is 16-bit PCM encoded `WAV <https://en.wikipedia.org/wiki/WAV>`_ files with an 8 kHz sample rate. 
-The Call Ingestion API supports `WAV <https://en.wikipedia.org/wiki/WAV>`_  and `MP3 <https://en.wikipedia.org/wiki/MP3>`_ file formats.  However, the Invoca Audio Processing system will upsample or downsample accordingly into our default call recording format.
+The Call Ingestion API supports `WAV <https://en.wikipedia.org/wiki/WAV>`_ , `MP3 <https://en.wikipedia.org/wiki/MP3>`_ and `MP4 <https://en.wikipedia.org/wiki/MP4_file_format>`_ file formats.  However, the Invoca Audio Processing system will upsample or downsample accordingly into our default call recording format.
```

Comment:
For MP4, we might want to note that the MP4's should be audio only (in other words, not video tracks on them). We can use the term "should", as they technically _can_, but ideally they won't.

It _might_ be easiest to do this by turning this into a bulleted list. E.g.

> The Call Ingestion API supports
>   * `WAV <https://en.wikipedia.org/wiki/WAV>`
>   * `MP3 <https://en.wikipedia.org/wiki/MP3>`
>   * `MP4 <https://en.wikipedia.org/wiki/MP4_file_format>`
>      * Note: MP4s should be audio-only


@ryanbales what are your thoughts here? worth mentioning in your opinion?

Reply:
cc @ryanbales ^

Reply:
@kamerynn sorry for the delay.  Def agree in calling out audio-only MP4s.  The layout you suggested above is 👍 for me!